include/fmt/base.h

```diff
@@ -279,6 +279,17 @@ import std;
 #  define FMT_UNICODE 1
 #endif
 
+// Check if rtti is available.
+#ifndef FMT_HAS_RTTI
```

Comment:
FMT_HAS_RTTI -> FMT_USE_RTTI to follow the usual convention and since it can be controlled by a user.